docs/make.jl

```diff
@@ -31,6 +32,7 @@ end
 
 example_pages = Any[
     "Thermal bubble" => "literated/thermal_bubble.md",
```

Comment:
```suggestion
```

Reply:
just because currently, the most thermal bubble includes a dry one?

Reply:
we could keep the stratified one just for now since its interesting, and that way we have 3 examples? and then we can remove it when we have a few more

Reply:
okie!